src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapBad.java

```diff
@@ -1,11 +1,13 @@
 package com.google.checkstyle.test. //warn
               chapter3filestructure.rule332nolinewrap;
 
-import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck; //ok
+import static java.math. //warn
+        BigInteger.ZERO;
 
-import javax.accessibility. //warn
-    AccessibleAttributeSequence;
+import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck; //ok
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater; //ok
+import javax.accessibility. //warn
+        AccessibleAttributeSequence;
```

Comment:
> please make a changes to this file as min as possible. You add two lines and all other are changed for no reason.

By changed do you mean all the lines after that are shifted by 2 or I made some other change i shouldn't have?

Also, regarding  @rnveach review, i need to add more tests in the `it` folder. I was thinking of something like 
1. renaming badLineWrapTest to `defaultBadLineWrapTest` 
 and goodLineWrapTest to `defaultGoodLineWrapTest`
2. creating new tests called `ignoreStaticImportBadLineWrapTest` and `ignoreStaticImportGoodLineWrapTest`, but using the `Configuration` class's object, I can't edit the attributes. Is it ok if I use the `DefaultConfiguration` class instead?